lib/Table.php

```diff
@@ -262,4 +263,17 @@ public function splitTableOnNewWorksheet(Sheet $sheet): self
 
         return $newTable;
     }
+
+    public function enableAutoFilter(): void
+    {
+        if (! $this->isEmpty() && $this->getDataRowStart() > 0) {
+            $minCol = $this->getColumnStart();
+            $minRow = $this->getDataRowStart() - 1; // header row
+            $maxCol = $this->getColumnEnd();
+            $maxRow = $this->getRowEnd();
+
+            $autoFilter = new AutoFilter($minCol, $minRow + 1, $maxCol, $maxRow + 1);
+            $this->activeSheet->setAutoFilter($autoFilter);
+        }
```

Comment:
The `if` condition needs to be inversed so we can test, and have code-coverage for, the case that `enableAutoFilter` gets called but nothing happens